demo/api/src/app.module.ts

```diff
@@ -66,23 +64,22 @@ import { RedirectScope } from "./redirects/dto/redirect-scope";
             }),
             inject: [configNS.KEY, BlocksTransformerService],
         }),
-        AuthModule.registerAsync({
+        AuthModule.register<CurrentUser>({
             imports: [ConfigModule],
-            useFactory: async (config: ConfigType<typeof configNS>) => ({
-                config: {
-                    apiPassword: config.API_PASSWORD,
-                    idpConfig: {
-                        url: config.IDP_API_URL,
-                        password: config.IDP_API_PASSWORD,
-                        clientId: config.IDP_CLIENT_ID,
-                    },
+            strategy: "authedUser",
+            useFactory: () => ({
+                authedUser: {
```

Comment:
Is this only used for local development? If not, how would an application get the current user?

Reply:
Mainly for local development, but it is also possible to use a static user in a deployed environment which is secured by other means. Authorization for production environments is using JWTs now: https://github.com/vivid-planet/comet/pull/903